scenarios/kubernetes_bazel.py

```diff
@@ -84,7 +84,15 @@ def query(kind, selected_pkgs, changed_pkgs):
 
     selection = '//...'
     if selected_pkgs:
-        selection = 'set(%s)' % ' '.join(selected_pkgs)
+        # packages without an operator prefix are implicitly additive
+        # when specifying build targets
+        selection = selected_pkgs[0]
+        for pkg in selected_pkgs[1:]:
+            if pkg.startswith('//'):
```

Comment:
it seems like this does the wrong thing if someone passes `--test=foo/... bar/...` instead of `--test=//foo/... //bar/...`?

Reply:
I could just match on the `-` prefix which is the only operator listed at https://docs.bazel.build/versions/master/command-line-reference.html#target-pattern-syntax
This also seems to suggest that target patterns only support `-`:
https://docs.bazel.build/versions/master/user-manual.html#target-patterns

Reply:
updated to only care about `-` and otherwise add `+`